README.md

```diff
@@ -276,81 +278,383 @@ schedule is listed as N/A.
 (\*The variable $`b`$ refers to the number of entries retrieved by the
 range lookup.)
 
-TODO: Document the average-case behaviour of lookups.
+#### Table Size <span id="performance_size" class="anchor"></span>
 
-#### In-memory size of tables <span id="performance_size" class="anchor"></span>
+The in-memory and the on-disk size of an LSM-tree scale *linearly* with
+the number of physical entries. However, the in-memory size is smaller
+by a significant factor. Let us look at a table that uses the default
+configuration and has 100 million entries with 34 byte keys and 60 byte
+values. The total size of 100 million key–value pairs is approximately
+8.75GiB. Hence, the on-disk size would be at least 8.75GiB, not counting
+the overhead for metadata.
 
-The in-memory size of an LSM-tree is described in terms of the variable
-$`n`$, which refers to the number of *physical* database entries. A
-*physical* database entry is any key–operation pair, e.g., `Insert k v`
-or `Delete k`, whereas a *logical* database entry is determined by all
-physical entries with the same key.
+TODO: Choose default Bloom filter size.
 
-The worst-case in-memory size of an LSM-tree is $`O(n)`$.
+The in-memory size would be approximately 0.99GiB:
 
-- The worst-case in-memory size of the write buffer is $`O(B)`$.
+- The write buffer would store at most 20,000 entries, which is
+  approximately 1.95MiB.
 
-  The maximum size of the write buffer on the write buffer allocation
-  strategy, which is determined by the `confWriteBufferAlloc` field of
-  `TableConfig`. Regardless of write buffer allocation strategy, the
-  size of the write buffer may never exceed 4GiB.
+- The fence-pointer indexes would store approximately 2.29 million keys,
+  which is approximately 74.41MiB.
 
-  `AllocNumEntries maxEntries`  
-  The maximum size of the write buffer is the maximum number of entries
-  multiplied by the average size of a key–operation pair.
+- The Bloom filters would use 9.85 bits per entry, which is
+  approximately 939.37MiB.
 
-- The worst-case in-memory size of the Bloom filters is $`O(n)`$.
+The in-memory size would be approximately 1.54GiB:
```

Comment:
I'm not following what we're presenting now. This seems to be a duplicate section with different numbers (16 bits per key?)

Reply:
I've left in a duplicate section depending on which default we settle on (1:100, 1:1000, 10b/e, 15b/e).